legacy/jacorb/src/main/java/org/jboss/as/jacorb/JacORBExtension.java

```diff
@@ -68,6 +70,8 @@
 
     private static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(1, 4, 0);
```

Comment:
1.4.0 is the EAP 6.4 version. This one is different, and thus it needs to be 2.0.0.

You'll then need to transform to 1.4.0. To save duplication, consider using ChainedTransformationDescriptionBuilder, e.g. 

https://github.com/wildfly/wildfly-core/blob/master/jmx/src/main/java/org/jboss/as/jmx/JMXExtension.java#L174

That's not required though; duplicating is ok.

Also, transforming to a ModelVersion prior to the one in EAP 6.2.0 is not required, so you can drop any of that code if you like.